Engines/Wine/QuickScript/GoG Script/script.js

```diff
@@ -0,0 +1,70 @@
+include(["engines", "wine", "quick_script", "quick_script"]);
+include(["utils", "functions", "net", "download"]);
+include(["engines", "wine", "engine", "object"]);
+
+function GogScript() {
+    QuickScript.call(this);
+}
+
+GogScript.prototype = Object.create(QuickScript.prototype);
+
+GogScript.prototype.constructor = GogScript;
+
+GogScript.prototype.gogSetupFileName = function(setupFileName) {
```

Comment:
Comments

Reply:
Done

Reply:
Are you sure, this method doesn't seem to have method comments as of now.
Maybe you mistook this method for `Downloader.prototype.json`?

Reply:
Done